### DISCUSSION TITLE
2022.10.22 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#129] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
이은혁 : 토이 프로젝트 crud 기능 구현하기
고영석 :
김지애 : 수업진도 따라가기, cs db 공부하기
김지오 : 1일 1 알고리즘 풀기 (재귀랑 dfs bfs), 수업진도 따라가기, 쟝고 복습하고 노션or 블로그에 정리
나혜승 : 자바스크립트 노션에 정리, api 세번 반복하기
이수연 : 점프 투 자바 끝내기, 1일 1알고리즘 문제풀이(요일별로 다른 알고리즘 분류 / pass 못해도 시도하기), 수업 진도 따라가기
이문삼 : 수업 진도 따라가기, CS 블로그에 포스팅하기, 자바스크립트로 알고리즘 풀기

**Comment by itmakesmesoft** [comment]
목표 : 토이 프로젝트 crud 기능 구현하기
점수 : 70점
이유 : 
기능은 구현 완료
But, Ssafy프로젝트와 다른점 무엇?
글고 api는 만들었는데,, 그다음은 어케할지 모르겠다...

**Comment by giokim12** [comment]
김지오 : 1일 1 알고리즘 풀기 (재귀랑 dfs bfs), 수업진도 따라가기, 쟝고 복습하고 노션or 블로그에 정리

1일 1 알고리즘 성공~ 재귀했어요
저는 드디어 현재를 살게 되었습니다. 하지만 과거가 더 행복했을지도
노션 정리는 아직 다는 안했지만 주말에 월말평가 대비하면서 다 하려고용

**Comment by HyeseungNA** [comment]
목표: 자바스크립트 노션에 정리, api 세번 반복하기
점수 : 65점
이유: 자바스크립트 배운 것들 노션에 정리 했고, api는 2번째 반복하고 있고 아직 안보고 할 수 없어서,,많이 부족합니다,,,,

**Comment by swjiae** [comment]
이번 주 목표 : 수업진도 따라가기, cs db 공부하기

점수 : 70점
이유 : 수업 진도 잘따라감, cs db 공부를 했지만 큰 틀만 보고 자세히는 못함

**Comment by YoungSeok222222** [comment]
이번주 목표: 인적성 공부하기, 진도 잘 따라가기, 블로그 글 3개 이상 올리기
점수: 5점

이유: 진도 따라가기와 인적성 공부하기 어느 것 하나 제대로 하지 못했고, 블로그 글도 올리지 못했기 때문에 반성의 의미로 5점을 줬습니다. 
 다음주 목표는 수업 내용 따라가기와 이전에 배웠던 내용 복습하기, 1일 1알고리즘 풀기, 블로그 글 3개 이상 올리기로 하겠습니다.

**Comment by ats166** [comment]
이번주 목표 : 1일 1알고 + 수업진도 따라가기
점수 : 60
이유 : 1일1알고는 했지만 A형 대비 하느라 진도를 잘못따라가서 다시 따라잡아야되는데 아직 못따라갔기때문에 60점을 줬습니다.

**Comment by Chung-HyeSu** [comment]
저번주 목표 : 수업 진도 따라가기, 운동, 컨디션 회복하기
점수 : 60 -> 이번주 컨디션이 너무 안좋았는데 이정도면 괜찮았다..
이번주 목표 : 프로그래머스 JS 풀기, sqld 공부, 블로그에 차근차근 글 올리기.
